responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -72,7 +72,8 @@ class ModelAnalysis(object):
 
     def __init__(self, model, train, test, target_column,
                  task_type, categorical_features=None, train_labels=None,
-                 serializer=None):
+                 serializer=None,
+                 maximum_rows_for_test: int = 5000):
```

Comment:
Can you do away with the spaces around the '='?

Reply:
I think spaces around the `=` is standard when adding type hints?

Reply:
Might be more clear if there are type hints on all params

Reply:
The space around the `=` is indeed a `flake8`ism. I agree that the type hints would be nice.... which is why I added one for the parameter I've added. The rest are.... out of scope for this.